templates/ansible/hieradata.j2

```diff
@@ -33,15 +33,12 @@ amazon_ssm_agent::proxy_url: {{ http_proxy }}
 
 aem_curator::install_author::aem_profile: {{ aem.profile }}
 aem_curator::install_author::aem_keystore_password: "%{lookup('{{ aem.keystore_password_parameter }}')}"
+aem_curator::install_author::aem_ssl_method: {{ aem.ssl_method }}
 
 aem_curator::install_author::aem_artifacts_base: {{ aem.artifacts_base }}
 aem_curator::install_author::aem_jvm_mem_opts: "{{ aem.author.jvm_mem_opts }}"
 aem_curator::install_author::aem_jvm_opts:
 - '-XX:+PrintGCDetails'
-- '-XX:+PrintGCTimeStamps'
-- '-XX:+PrintGCDateStamps'
-- '-XX:+PrintTenuringDistribution'
-- '-XX:+PrintGCApplicationStoppedTime'
 - '-XX:+HeapDumpOnOutOfMemoryError'
```

Comment:
this is only required for jdk11 right ? I guess best is to make those parameters configurable, so the user can adjust them to their own need. Though these parameters represent the default values.

Reply:
It is needed for JDK11, I will make this change.

Reply:
Done